notebook/Dockerfile

```diff
@@ -39,6 +39,9 @@ RUN mamba install -y \
     ipywidgets \
     cachey \
     streamz \
+    notebook \
+    jupyterhub \
```

Comment:
I'm not too concerned about `notebook` as this gets installed with `jupyterlab`, however I'm wondering if `jupyterhub` is needed here. Was this added for compatibility with the previous `base-notebook` image?

Reply:
Looks like [`jlpm` comes from `jupyterlab`]( https://github.com/conda-forge/jupyterlab-feedstock/blob/fa4de04088e34bd72e0a3ee6a33fd5737d597acc/recipe/meta.yaml#L19 )

Reply:
Yeah, looking back at the logs it looks like the issue isn't with the fact that `jlpm` isn't installed, but that Node.js isn't installed, which causes `jlpm` to error when we attempt to clean the cache.

I did some digging in the generated container and it didn't seem like there was anything to be cleaned up by `jlpm`, so just went ahead and removed that cleanup step. If there are specific folders we know we want to remove to trim the image size maybe it would make sense to `rm -rf` them explicitly rather than using `jlpm` so that we can avoid having to install the `nodejs` package, which I imagine would bloat things up a decent amount